plugins/ltac/g_tactic.mlg

```diff
@@ -107,11 +86,10 @@ let check_for_coloneq =
       | _ -> err ())
 
 let lookup_at_as_comma =
-  Pcoq.Entry.of_parser "lookup_at_as_comma"
-    (fun _ strm ->
-      match stream_nth 0 strm with
-        | KEYWORD (","|"at"|"as") -> ()
-        | _ -> err ())
+  let open Pcoq.Lookahead in
+  entry_of_lookahead "lookup_at_as_comma" begin
+    lk_kw "," <+> lk_kw "at" <+> lk_kw "as"
```

Comment:
Use `lk_kws` instead.

Reply:
Right, I introduced it after and forgot to come back to this example. Fixed.